openmc/universe.py

```diff
@@ -111,6 +111,12 @@ def create_xml_subelement(self, xml_element, memo=None):
 
         """
 
+    def _deepcopy_universe_for_clone(self):
+        """Deepcopy an openmc.UniverseBase object. This is a paceholder for any future classes inherited from this one.
+        This should only to be used within the openmc.UniverseBase.clone() context.
+        """
+        return deepcopy(self)
```

Comment:
Rather than this approach, I would suggest making this an `@abstractmethod` that is required to be overridden by a subclass. You can look at the `create_xml_subelement` method for an example of this.

Reply:
Great call!